.github/workflows/fossa.yml

```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.14.x"
```

Comment:
Not sure if important/relevant. Should we _go_ with 1.15? :wink:

Reply:
:-) Let's fix it in post (so we can get Ihor's PR merged and not ask more of them)